pom.xml

```diff
@@ -1,336 +1,336 @@
-												</format>
-											</contributors>
-											<hide>
-												<contributors>
-													<contributor>bot</contributor>
-													<contributor>Bot</contributor>
-													<contributor>GitHub</contributor>
-												</contributors>
-											</hide>
-										</changelog>
-									</github>
-								</release>
-								<signing>
-									<active>ALWAYS</active>
-									<armored>true</armored>
-								</signing>
-								<deploy>
-									<maven>
-										<nexus2>
-											<maven-central>
-												<active>ALWAYS</active>
-												<url>https://s01.oss.sonatype.org/service/local</url>
-												<snapshotUrl>
-													https://s01.oss.sonatype.org/content/repositories/snapshots/
-												</snapshotUrl>
-												<closeRepository>true</closeRepository>
-												<releaseRepository>true</releaseRepository>
-												<stagingRepositories>target/staging-deploy</stagingRepositories>
-											</maven-central>
-										</nexus2>
-									</maven>
-								</deploy>
-							</jreleaser>
-						</configuration>
-					</plugin>
-				</plugins>
-			</build>
-		</profile>
-	</profiles>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>dev.cdevents</groupId>
+	<artifactId>cdevents-sdk-java-parent</artifactId>
+	<version>0.3.0-SNAPSHOT</version>
+	<packaging>pom</packaging>
+
+	<name>cdevents-sdk-java-parent</name>
+	<description>CDEvents Java SDK</description>
+	<url>https://github.com/cdevents</url>
+	<inceptionYear>2022</inceptionYear>
+
+	<organization>
+		<name>Continuous Delivery Foundation</name>
+		<url>https://cd.foundation</url>
+	</organization>
+
+	<properties>
+		<project.identifier>${project.name}</project.identifier>
+		<maven.compiler.source>8</maven.compiler.source>
+		<maven.compiler.target>8</maven.compiler.target>
+		<maven.compiler.release>8</maven.compiler.release>
+		<cloudevents.version>2.3.0</cloudevents.version>
+		<junit-jupiter.version>5.8.2</junit-jupiter.version>
+		<assertj-core.version>3.22.0</assertj-core.version>
+		<jackson.version>2.16.1</jackson.version>
+		<slf4j.version>2.0.9</slf4j.version>
+		<json.schema.version>1.3.2</json.schema.version>
+		<packageurl.version>1.4.1</packageurl.version>
+		<commons-lang3.version>3.14.0</commons-lang3.version>
+		<project.github.repository>cdevents/sdk-java</project.github.repository>
+		<nexus.url>https://s01.oss.sonatype.org</nexus.url>
+		<repository.url>git@github.com:${project.github.repository}.git</repository.url>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<project.build.outputTimestamp>2024-02-29T14:23:24Z</project.build.outputTimestamp>
```

Comment:
Please change the value of this property to the following expression: `${git.commit.author.time}`

Reply:
Done